assets/eip-7092/BondStorage.sol

```diff
@@ -1,33 +1,27 @@
 // SPDX-License-Identifier: CC0-1.0
-pragma solidity ^0.8.18;
+pragma solidity ^0.8.0;
 
 contract BondStorage {
     struct Bond {
         string isin;
         string name;
         string symbol;
         address currency;
-        address currencyOfCoupon;
```

Comment:
@Pandapip1 ,

Some fields are removed because the example uses only parameters that are REQUIRED. Therefore,  OPTIONAL params have been removed